src/main/java/propertyFiles/Bundle.properties

```
@@ -503,6 +504,7 @@ harvestclients.newClientDialog.step4=Step 4 of 4 - Display
 harvestclients.newClientDialog.harvestingStyle=Archive Type
 harvestclients.newClientDialog.harvestingStyle.tip=Type of remote archive.
 harvestclients.newClientDialog.harvestingStyle.helptext=Select the archive type that best describes this remote server in order to properly apply formatting rules and styles to the harvested metadata as they are shown in the search results. Note that improperly selecting the type of the remote archive can result in incomplete entries in the search results, and a failure to redirect the user to the archival source of the data.
+harvestclients.newClientDialog.harvestingStyle.required=Please select one of the values from the menu
```

```suggestion
harvestclients.newClientDialog.harvestingStyle.required=Please select one of the values from the menu.
```

I didn't run the code to see how the GUI looks but perhaps this sentence should end with a period? Apologies if I'm mistaken and some other string is concatenated or something.

---

ok, done.